server/src/main/java/org/opensearch/discovery/InitializeExtensionsRequest.java

```diff
@@ -25,52 +24,49 @@
  */
 public class InitializeExtensionsRequest extends TransportRequest {
     private final DiscoveryNode sourceNode;
-    /*
-     * TODO change DiscoveryNode to Extension information
-     */
-    private final List<DiscoveryExtension> extensions;
+    private final DiscoveryExtension extension;
```

Comment:
The initial intention for this was to help the extension understand which extensions are installed in the system. 
If the dependencies of extension being initialized don't exist, it should fail the init. 

I love this simple approach not thinking too far ahead, @dbwiddis could you open an issue to design how can extensions discover if its dependencies exist.

Reply:
Exist and are initialized.... something the orchestrator knows... but we don't yet have a "depends on" field in extensions or which field (uniqueID?) is used to ID that other extension.  I'll open a tracking issue as still many unknowns there.